diff --git a/node_modules/react-native-maps/src/MapCallout.tsx b/node_modules/react-native-maps/src/MapCallout.tsx
index 54a2b72..92495ab 100644
--- a/node_modules/react-native-maps/src/MapCallout.tsx
+++ b/node_modules/react-native-maps/src/MapCallout.tsx
@@ -45,9 +45,6 @@ export class MapCallout extends React.Component<MapCalloutProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   render() {
     const {tooltip = false, alphaHitTest = false} = this.props;
diff --git a/node_modules/react-native-maps/src/MapCalloutSubview.tsx b/node_modules/react-native-maps/src/MapCalloutSubview.tsx
index 32d10ec..f8929b8 100644
--- a/node_modules/react-native-maps/src/MapCalloutSubview.tsx
+++ b/node_modules/react-native-maps/src/MapCalloutSubview.tsx
@@ -30,9 +30,6 @@ export class MapCalloutSubview extends React.Component<MapCalloutSubviewProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
   render() {
     const AIRMapCalloutSubview = this.getNativeComponent();
     return (
diff --git a/node_modules/react-native-maps/src/MapCircle.tsx b/node_modules/react-native-maps/src/MapCircle.tsx
index ad03aa3..c7c04b2 100644
--- a/node_modules/react-native-maps/src/MapCircle.tsx
+++ b/node_modules/react-native-maps/src/MapCircle.tsx
@@ -125,9 +125,6 @@ export class MapCircle extends React.Component<MapCircleProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   private circle: NativeProps['ref'];
 
diff --git a/node_modules/react-native-maps/src/MapHeatmap.tsx b/node_modules/react-native-maps/src/MapHeatmap.tsx
index 0dc3112..803bcc5 100644
--- a/node_modules/react-native-maps/src/MapHeatmap.tsx
+++ b/node_modules/react-native-maps/src/MapHeatmap.tsx
@@ -89,9 +89,6 @@ export class MapHeatmap extends React.Component<MapHeatmapProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   private heatmap: NativeProps['ref'];
 
diff --git a/node_modules/react-native-maps/src/MapLocalTile.tsx b/node_modules/react-native-maps/src/MapLocalTile.tsx
index 8911dee..b12fbc3 100644
--- a/node_modules/react-native-maps/src/MapLocalTile.tsx
+++ b/node_modules/react-native-maps/src/MapLocalTile.tsx
@@ -42,9 +42,6 @@ export class MapLocalTile extends React.Component<MapLocalTileProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   render() {
     const AIRMapLocalTile = this.getNativeComponent();
diff --git a/node_modules/react-native-maps/src/MapMarker.tsx b/node_modules/react-native-maps/src/MapMarker.tsx
index 39d9e5b..7602982 100644
--- a/node_modules/react-native-maps/src/MapMarker.tsx
+++ b/node_modules/react-native-maps/src/MapMarker.tsx
@@ -373,9 +373,6 @@ export class MapMarker extends React.Component<MapMarkerProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   static Animated: Animated.AnimatedComponent<typeof MapMarker>;
 
diff --git a/node_modules/react-native-maps/src/MapOverlay.tsx b/node_modules/react-native-maps/src/MapOverlay.tsx
index 1db5d07..5660404 100644
--- a/node_modules/react-native-maps/src/MapOverlay.tsx
+++ b/node_modules/react-native-maps/src/MapOverlay.tsx
@@ -88,9 +88,6 @@ export class MapOverlay extends React.Component<MapOverlayProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   static Animated: Animated.AnimatedComponent<typeof MapOverlay>;
 
diff --git a/node_modules/react-native-maps/src/MapPolygon.tsx b/node_modules/react-native-maps/src/MapPolygon.tsx
index a63e86a..968bb7e 100644
--- a/node_modules/react-native-maps/src/MapPolygon.tsx
+++ b/node_modules/react-native-maps/src/MapPolygon.tsx
@@ -151,9 +151,6 @@ export class MapPolygon extends React.Component<MapPolygonProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   private polygon: NativeProps['ref'];
 
diff --git a/node_modules/react-native-maps/src/MapPolyline.tsx b/node_modules/react-native-maps/src/MapPolyline.tsx
index 20d9981..071b423 100644
--- a/node_modules/react-native-maps/src/MapPolyline.tsx
+++ b/node_modules/react-native-maps/src/MapPolyline.tsx
@@ -152,9 +152,6 @@ export class MapPolyline extends React.Component<MapPolylineProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   private polyline: NativeProps['ref'];
 
diff --git a/node_modules/react-native-maps/src/MapUrlTile.tsx b/node_modules/react-native-maps/src/MapUrlTile.tsx
index 586daff..5779394 100644
--- a/node_modules/react-native-maps/src/MapUrlTile.tsx
+++ b/node_modules/react-native-maps/src/MapUrlTile.tsx
@@ -151,9 +151,6 @@ export class MapUrlTile extends React.Component<MapUrlTileProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   render() {
     const AIRMapUrlTile = this.getNativeComponent();
diff --git a/node_modules/react-native-maps/src/MapWMSTile.tsx b/node_modules/react-native-maps/src/MapWMSTile.tsx
index 17eca2f..6b31a04 100644
--- a/node_modules/react-native-maps/src/MapWMSTile.tsx
+++ b/node_modules/react-native-maps/src/MapWMSTile.tsx
@@ -130,9 +130,6 @@ export class MapWMSTile extends React.Component<MapWMSTileProps> {
   // declaration only, as they are set through decorateMap
   /// @ts-ignore
   context!: React.ContextType<typeof ProviderContext>;
-  getNativeComponent!: () => NativeComponent<NativeProps>;
-  getMapManagerCommand!: (name: string) => MapManagerCommand;
-  getUIManagerCommand!: (name: string) => UIManagerCommand;
 
   render() {
     const AIRMapWMSTile = this.getNativeComponent();
